Automattic/woocommerce-payments
# Fix test mode notice to match Page width

Fixes #641

#### Changes proposed in this Pull Request

* Refactor test notice into a regular component.
* Add test notice to all pages.
* Refactor test-mode-notice tests.

#### Testing instructions

* Make sure you have dev mode enabled.
* Navigate to the following pages, and check if the test notice matches the width of the content below it:
  * Payments > Deposits
  * Payments > Deposits > Deposit Details
  * Payments > Transactions
  * Payments > Transactions > Payment Details
  * Payments > Disputes
  * Payments > Disputes > Dispute Details
  * Payments > Disputes > Challenge Dispute

#### Screenshots

Before:

![before](https://user-images.githubusercontent.com/5509901/104652553-068f3b80-5698-11eb-9431-8f6445bef7dc.png)

After:

![after](https://user-images.githubusercontent.com/5509901/104652561-0b53ef80-5698-11eb-9325-09c6eaf6067b.png)


-------------------
